docs/source/markdown/podman-mount.1.md.in

```diff
@@ -12,12 +12,11 @@ podman\-mount - Mount a working container's root filesystem
 Mounts the specified containers' root file system in a location which can be
 accessed from the host, and returns its location.
 
-If you execute the command without any arguments, Podman will list all of the
-currently mounted containers, including external containers. External containers are
-containers in container/storage by tools other than Podman. For example Buildah and
-CRI-O.
+Podman list all of the currently mounted containers, including external containers, when the command is executed
```

Comment:
**lists**, with **s**, not list. The standalone "For example" sentence bothers me a little, but it was there in the original and I'm getting tired so I'll pretend I don't see it.